CHANGELOG.md

```diff
@@ -5,6 +5,121 @@ information on the list of deprecated flags and APIs please have a look at
+- Fix behavior of absolute paths in .dockerignore [moby/moby#32088](https://github.com/moby/moby/pull/32088)
+- Fix Cache with ONBUILD [moby/moby#33524](https://github.com/moby/moby/pull/33524) [moby/moby#33500](https://github.com/moby/moby/pull/33500)
+* Refactor builder probe cache and container backend [moby/moby#33061](https://github.com/moby/moby/pull/33061)
+
+### Distribution
+
+* Stop trying to load images on an incompatible OS [moby/moby#33647](https://github.com/moby/moby/pull/33647)
+* Do not add duplicate platform information to service spec [moby/moby#33867](https://github.com/moby/moby/pull/33867)
+
+### Logging
+
+- Fix log readers can block writes indefinitely [moby/moby#33897](https://github.com/moby/moby/pull/33897)
++ Add stream format details for attach/logs endpoint [moby/moby#33494](https://github.com/moby/moby/pull/33494)
+- Fix wrong filemode for rotate log files [moby/moby#33926](https://github.com/moby/moby/pull/33926)
+* Expose config credentials without needing the Cli [docker/cli#227](https://github.com/docker/cli/pull/227)
+* System prune: only warn for volumes if --volumes is given [docker/cli#255](https://github.com/docker/cli/pull/255)
+- Fix stderr logging for journald and syslog [moby/moby#33832](https://github.com/moby/moby/pull/33832)
+* Pass the same log filename to logging plugins on start/stop requests [moby/moby#33583](https://github.com/moby/moby/pull/33583)
+
+### Networking
+
+* Include Endpoint List for Shared Endpoints [moby/moby#33662](https://github.com/moby/moby/pull/33662)
+* LCOW: owner and network endpoints [moby/moby#33774](https://github.com/moby/moby/pull/33774)
+* Host and Bridge network support in docker stack deploy [docker/cli#132](https://github.com/docker/cli/pull/132)
+
+### Runtime
+
+* Keep pause state when restoring container's status [moby/moby#34071](https://github.com/moby/moby/pull/34071)
+* Plugable secret backend [moby/moby#34157](https://github.com/moby/moby/pull/34157) [moby/moby#34123](https://github.com/moby/moby/pull/34123)
+* Small refactor in system prune [docker/cli#341](https://github.com/docker/cli/pull/341)
+* Store container names in memdb [moby/moby#33886](https://github.com/moby/moby/pull/33886)
+* In the case of remounting with changed data, need to call mount [moby/moby#34077](https://github.com/moby/moby/pull/34077)
+* LCOW: Service VM lifetime changes [moby/moby#33969](https://github.com/moby/moby/pull/33969)
+* Enable inspect exec if container is pause/restarting/non-running [moby/moby#32881](https://github.com/moby/moby/pull/32881)
+* Allow stopping of paused container [moby/moby#34027](https://github.com/moby/moby/pull/34027)
+* Devicemapper: rework logging and add --storage-opt dm.libdm_log_level [moby/moby#33845](https://github.com/moby/moby/pull/33845)
+* Devicemapper: Wait for device removal if deferredRemoval=true and deferredDeletion=… [moby/moby#33877](https://github.com/moby/moby/pull/33877)
```

Comment:
Devicemapper: Prevent "device in use" errors if deferred removal is enabled, but not deferred deletion